### DISCUSSION TITLE
Running Strimzi Without ClusterRole, Cluster level permissions

### DISCUSSION THREAD

**Discussion by ghost** [strimzi/strimzi-kafka-operator#4720] [Q&A]
Hey,

Installed Strimzi on kubernetes and found it requires cluster operator. Unfortunately, this has raised concerns + almost voiding the usage of it. The reason being there are other projects running on the cluster in other namespaces, don't want to open up / grant cluster permission to strimzi (in a nice way).

Is there a way to run strimzi without it requiring cluster level resources / cluster role binding?

I noticed company likes Redhat use strimzi - surely there must be a way? Can we change it to role binding - will it work?

**Comment by scholzj** [comment]
The cluster roles are there not just for fun - they are there because functionality which is needed for production-grade Kafka such as rack-awareness and some slightly less critical such as volume resizing or access using node-ports. If you do not want to give the operator the rights to read node addresses or storage classes and do not need the features depending on it, you can run it without the ClusterRoleBindings.

**Reply by ppaslan** [reply]
Is the information above still accurate because we are in the same situation as the user above.
Is there a list of features which would stop working if one decided to go with the createGlobalResources: false option?

**Reply by scholzj** [reply]
Assuming you talk about the `createGlobalResources: false` option in the Helm Chart ... that expects you to create the resources out of band. E.g. have them created by someone else already. It does not configure the operator to not need them.

In general, it is as described before -> you have to create the CRDs and ClusterRoles. You can change the ClusterRoleBindings to RoleBindings - but you should not use the features which require these rights such as rack awareness, node ports or storage resizing.
